PokeAlarm/Events/EggEvent.py

```diff
@@ -79,6 +79,14 @@ def generate_dts(self, locale, timezone, units):
             '12h_raid_end': raid_end_time[1],
             '24h_raid_end': raid_end_time[2],
 
+            # Time Remaining Without Seconds
+            'hatch_time_left_minutes': hatch_time[3],
```

Comment:
I don't like minutes, because it implies that the other doesn't have minutes. How about `hatch_time_no_secs`?